frontend/mulighetsrommet-veileder-flate/src/core/api/queries/useHentDeltMedbrukerStatus.ts

```diff
@@ -5,27 +5,28 @@ import { mulighetsrommetClient } from "../clients";
 import { QueryKeys } from "../query-keys";
 import { useHentVeilederdata } from "./useHentVeilederdata";
 
-export function useHentDeltMedBrukerStatus(sanityId: string | undefined, norskIdent: string) {
+export function useHentDeltMedBrukerStatus(norskIdent: string, id?: string) {
   const { data: veilederData } = useHentVeilederdata();
 
-  const requestBody = {
-    norskIdent,
-    sanityId: sanityId ?? "",
-  };
-
   const { data: sistDeltMedBruker, refetch: refetchDelMedBruker } = useQuery<DelMedBruker>(
-    [QueryKeys.DeltMedBrukerStatus, norskIdent, sanityId],
-    () => mulighetsrommetClient.delMedBruker.getDelMedBruker({ requestBody }),
-    { enabled: !erPreview() && !!sanityId },
+    [QueryKeys.DeltMedBrukerStatus, norskIdent, id],
+    () =>
+      mulighetsrommetClient.delMedBruker.getDelMedBruker({ requestBody: { norskIdent, id: id!! } }),
+    { enabled: !erPreview() && !!id },
   );
 
-  async function lagreVeilederHarDeltTiltakMedBruker(dialogId: string, sanityId: string) {
+  async function lagreVeilederHarDeltTiltakMedBruker(
+    dialogId: string,
+    tiltaksgjennomforingId?: string,
```

Comment:
Hadde implementasjon blitt enklere om dette funksjonen tok inn `tiltaksgjennomforing: Tiltaksgjennomforing` i stedet for to id'er?